src/config.js

```diff
@@ -0,0 +1,44 @@
+const os = require('os');
+const path = require('path');
+const fs = require('fs');
+
+const configName = '.todolor.conf';
+const defaultPath = '.todolor';
+
+// TODO: Write a small util to paint text
+const yellowAnsi = '\x1b[33m';
+const defaultAnsi = '\x1b[39m';
+
+const parse = (contents) => {
+  // TODO: Rewrite without regexp
+  const regExp = /PATH=(.*)/;
+  const result = regExp.exec(contents);
+  if (!result || result.length === 0) {
+    throw Error('Corrupted configuration file');
+  } else {
+    return result[0];
+  }
+};
+
+const createConfig = (configPath) => {
+  const databasePath = path.join(os.homedir(), defaultPath);
+  const entry = `PATH=${databasePath}`;
+  fs.writeFileSync(configPath, entry);
+  return databasePath;
+};
```

Comment:
Decompose this function to make it more readable.

Reply:
I thought about it, the problem is not with readability as such, but with the strange return, which works poorly with the function's name. Will definitely improve it after unit tests are written.